srcpkgs/ctags/template

```diff
@@ -1,14 +1,21 @@
 # Template file for 'ctags'
 pkgname=ctags
-version=5.8
-revision=7
+version=5.9.20210214.0
+revision=1
+wrksrc=ctags-p${version}
 build_style=gnu-configure
+hostmakedepends="pkg-config automake python3-docutils"
 short_desc="Generates an index file of language objects found in source files"
 maintainer="Jan S. <jan.schreib@gmail.com>"
-license="GPL-3"
-homepage="http://ctags.sourceforge.net/"
-distfiles="${SOURCEFORGE_SITE}/ctags/${version}/ctags-${version}.tar.gz"
-checksum=0e44b45dcabe969e0bbbb11e30c246f81abe5d32012db37395eb57d66e9e99c7
+license="GPL-2.0-only"
+homepage="https://ctags.io/"
+distfiles="https://github.com/universal-ctags/ctags/archive/p${version}.tar.gz"
+checksum=26d9efe481a57e81195878cdb0baca730a3c4d234b9a7c4f6bf2c8def42bf42b
+
+do_configure() {
+	./autogen.sh
```

Comment:
You can do `autogen.sh` in `pre_configure`, which is preferred. I think perl in host might make sense as well.

This depends on the resolution to https://github.com/void-linux/void-packages/pull/28544#issuecomment-782339333

Reply:
> You can do autogen.sh in pre_configure, which is preferred. I think perl in host might make sense as well.

understood

> This depends on the resolution to #28544 (comment)

So are we trying to disable checks ? all checks ? or add a patch to disable some specifcs checks ?

Or maybe I'm not aware of enough github issues discussions :D

Reply:
> I think perl in host might make sense as well.

Why ?

Reply:
Because it can be used in autoreconf... If it isn't necessary, we can keep it out.

The commit message should include that you are moving to a different upstream.

Reply:
> If it isn't necessary, we can keep it out.

It seems completely ok without. Without additional reasons, I dont think we should add it